### DISCUSSION TITLE
Help/ Guidance needed - I would like to customize the Lighthouse Generated Report

### DISCUSSION THREAD

**Discussion by sajankota** [GoogleChrome/lighthouse#15242] [General] [answered]
Hello Community,

I would like to customize the Lighthouse Generated Report HTML looks. Where do I start? Any guidance would be highly appreciated.

**Comment by adamraine** [comment, marked as answer]
Most of our report code is in https://github.com/GoogleChrome/lighthouse/tree/main/report. Feel free to start there.
